### PR TITLE
[Lib] Upgrade react-spring@8.0.23

### PR DESCRIPTION
**Pull request by damassi**
Upgrades react spring to latest, and adds a few more storybooks entries. 

Verified things are still working in Emission: 

expand

And here's palette: 

chart

### CODE REVIEW

File: packages/palette/src/elements/DonutChart/DonutChart.tsx [hunk 1]
```diff
@@ -147,12 +151,18 @@ export const DonutChart: React.FC<DonutChartProps> = ({
...
   )
 
+  const ChartSVG = () => (
+    <>
+      {tooltip}
+      {labels}
+      {svg}
+    </>
+  )
+
```
Comment (damassi) [inline packages/palette/src/elements/DonutChart/DonutChart.tsx]: This is leading to some strangeness in regards to `hasEnteredViewport`. Would be worth playing with it a bit.

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 3c8c9a91c5f30e98b89ada90807db6faaa7c8fb3

https://deploy-preview-511--artsy-palette.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 21f100acf819b946926553d4c8505949d9154249

https://deploy-preview-511--artsy-palette.netlify.com

**Comment by artsyit** [comment]
:rocket: PR was released in v4.14.14 :rocket:

File: packages/palette/src/elements/DonutChart/DonutChart.tsx (hunk 1 above)
Comment (ds300) [reply packages/palette/src/elements/DonutChart/DonutChart.tsx]: Why did you change this? You're creating a new react component type on every render here, which makes React unable to track changes properly.

Comment (damassi) [reply packages/palette/src/elements/DonutChart/DonutChart.tsx]: This was just an experiment to see the animation appear on load, nothing definitive. @sepans is on it.

**Pull request merged**